docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: '3.8'
```

Comment:
one more silly question from me, what does this version refer to?

Reply:
The version field in docker-compose.yml has been removed because:

- It is obsolete in Docker Compose V2
- Docker Compose now automatically uses the appropriate version
- Removing it eliminates warning messages during build/run
- It follows the current best practices for Docker Compose files